tests/snmp/test_snmp_queue_counters.py

```diff
@@ -24,6 +24,15 @@ def get_queue_ctrs(duthost, cmd):
     return len(duthost.shell(cmd)["stdout_lines"])
 
 
+def get_queuestat_ctrs(duthost, cmd):
```

Comment:
Hi @harjotsinghpawra , have you tested the logic on single ASIC devices? 
Does it also work fine?